content/handbook/engineering/devops/plan/product-planning/how-we-work.md

```
@@ -1,246 +1,71 @@
+| **Issue** | 1つのマージリクエスト、1人、1つのカテゴリー（FEまたはBE） | 1週間未満 | "ボードカードコンポーネントを追加"、"保存ビューAPI用の`filter`クエリパラメーターを作成" |
+| **Epic** | 複数のIssueをグループ化するスコープ化された成果物 | 1週間から3マイルストーン | "保存ビューでのBoards Alpha"、"ワークアイテムカード" |
+| **Task** | Issue内のチェックリスト項目/リマインダー | — | "changelogを更新"、"ドキュメントを更新" |
 
-## 方向性項目
+より大きなプロジェクトでワークアイテムタイプをどう使うかについては、[プロジェクト例: Boards](./project-example-boards.md)を参照してください。私たちはウェイトを使用しません。
 
-顧客向けの成果物であり、高インパクトな項目には ~"direction" のラベルが付きます。これらの項目はリリースカットオフの2日前までに本番環境に入れ、検証のための十分な時間を確保するよう努めています。
+### 良いIssueとは
 
-### 重大度の高い Issue
+- 1人のエンジニアが1週間以下で完了できるほど小さい。それより大きい場合は、エピックに昇格するか、[スパイク](#spikes-and-proof-of-concepts)を作成して分解することを検討します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)